cocos/3d/CCTerrain.h

```diff
@@ -476,11 +479,13 @@ class CC_DLL Terrain : public Node
     float _lodDistance[3];
     Texture2D * _detailMapTextures[4];
     Texture2D * _alphaMap;
+    Texture2D * _lightMap;
+    Vec3 _lightDir;
     CustomCommand _customCommand;
     QuadTree * _quadRoot;
     Chunk * _chunkesArray[MAX_CHUNKES][MAX_CHUNKES];
     std::vector<TerrainVertexData> _vertices;
-    std::vector<GLushort> _indices;
+    std::vector<unsigned int> _indices;
```

Comment:
It seems that `unsigned int index` could not work correctly on some old android device?
Could we keep using GLushort?